catalog/spatial/registry/registry-federation-admin-impl/src/main/java/org/codice/ddf/registry/federationadmin/impl/AdminHelper.java

```diff
@@ -38,7 +43,13 @@
             ConfigurationAdmin.SERVICE_FACTORYPID,
             ConfigurationAdmin.SERVICE_FACTORYPID);
 
-    private static final String MAP_ENTRY_ID = "id";
+    private static final String REGISTRY_POLICY_PID = "org.codice.ddf.registry.policy.RegistryPolicyPlugin";
```

Comment:
❓ Did you run the formatter? My spider sense tells me this should be two lines...